src/containers/login/index.tsx

```diff
@@ -37,43 +42,37 @@ const LoginPageContainer = styled.div`
   width: 80vw;
 `;
 
-const TabletLoginPageContainer = styled.div`
-  padding: 90px 60px;
-`;
-
 const MobileLoginPageContainer = styled.div`
   padding: 30px;
 `;
 
-const CenterDiv = styled.div`
-  display: flex;
-  justify-content: center;
-`;
-
-const RightMargin = styled.div`
-  margin-right: 30px;
-`;
-
-const InputContainer = styled(Col)`
-  height: 481px;
-  min-width: 250px;
-  padding: 30px 20px 20px 50px;
+export const TabletInputContainer = styled.div`
```

Comment:
❓ There are pretty small differences between this container and the tablet container on the login page, can they be abstracted without changing styling too much?

Reply:
I think it's better to keep them separate - the differences are in height/margin, where the signup needs to have a larger height and smaller margins because its form has more fields and is more sensitive to changes in browser height. We could make the login taller without impacting usability, but it adds a lot of blank space. Mostly a design choice.